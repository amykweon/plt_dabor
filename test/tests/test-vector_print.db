vector x;

x = horizontal (1);
print_vector x;
x = vertical (1);
print_vector x;
x = diagonalLeft (1);
print_vector x;
x = diagonalRight (1);
print_vector x;