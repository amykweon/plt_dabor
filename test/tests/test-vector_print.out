Hori (1)
Vert (1)
DiagL (1)
DiagR (1)